docs: updated README.md and  updated contribution.md (#17)
dscrait/dscrait.tech: contribution.md

* Showing empathy towards other community members

By participating, you are expected to uphold this code. Please report unacceptable behavior to [dsc.rait@gmail.com](dsc.rait@gmail.com).

If chat is more your speed, you can join the DSC-RAIT Discord Channel:

* [Join the DSC-RAIT Discord Channel](https://discord.gg/H8wTXrm)
    * Even though Discord is a chat service, sometimes it takes few hours for community members to respond &mdash; please be patient!
   

## What should I know before I get started?

dscrait.tech is a open source project &mdash; it's made up of [repositories](https://github.com/dscrait/dscrait.tech). When you initially consider contributing to 
dscrait.tech, 
you might be unsure about which of those repositories implements the functionality you want to change or report a bug for. This section should help you with that.

## How Can I Contribute?

**1.** Fork [this](https://github.com/dscrait/dscrait.tech.git) repository.

**2.** Clone the forked repository.
```terminal
git clone https://github.com/dscrait/dscrait.tech.git
```

**3.** Navigate to the project directory.
```terminal
cd dscrait.tech
```

**4.** Install NPM packages.
```terminal

> npm install

Reviewer: ```suggestion
yarn
```

```
**5.** Run the website in your localhost by typing this command.
```terminal

> npm run dev

Reviewer: yarn run dev

# Contributing to dscrait.tech

First off, thanks for taking the time to contribute! 

> The following is a set of guidelines for contributing to dscrait.tect, which is hosted in the [dscrait.tech](https://dscrait.tech). These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.

Reviewer: ```suggestion
The following is a set of guidelines for contributing to dscrait.tech, which is hosted in the [dscrait.tech](https://dscrait.tech). These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document by opening up an issue.
```

## How Can I Contribute?

**1.** Fork [this](https://github.com/dscrait/dscrait.tech.git) repository.

**2.** Clone the forked repository.
```terminal
git clone https://github.com/dscrait/dscrait.tech.git
```

**3.** Navigate to the project directory.
```terminal
cd dscrait.tech
```

**4.** Install NPM packages.
```terminal
npm install
```
**5.** Run the website in your localhost by typing this command.
```terminal
npm run dev
```

**6.** Create a new Branch.
```terminal
git branch <your_branch_name>
```

**7.** Switched to your new branch.
```terminal
git checkout -b <your_branch_name>
```

**8.** Make changes locally.
```terminal
git add filename.md  
```

**9.** Before commiting your changes add the following commands on your terminal
```terminal 
git remote add upstream https://github.com/dscrait/dscrait.tech.git
```

```terminal
git pull upstream master
```

**10.** Commit your changes.

```terminal
  git add .
  git commit -m "<your_commit_message>"
  
  # If so try to use conventional commit messages using the guide: https://www.conventionalcommits.org/en/v1.0.0/
```

**11.** Push your local branch to the remote repository.
```terminal

> git push 

Reviewer: git push origin <branch-name>

Author: Okay I will make the required changes, let me know if any thing else to change!!

```

**12.** Create a Pull Request! 

### Your First Code Contribution

Unsure where to begin contributing to dscrait.tech? You can start by looking through these `good first issues`:

* Good First issues - issues which should only require a few lines of code, and a test or two.

## Styleguides

> ### Git Commit Messages

Reviewer: add commit message conventions are already added above: https://www.conventionalcommits.org/en/v1.0.0/

Reviewer: Remove this section
